src/org/opensolaris/opengrok/web/api/v1/controller/SearchController.java

```diff
@@ -58,7 +61,7 @@ public SearchResult search(
             @QueryParam("path") final String path,
             @QueryParam("hist") final String hist,
             @QueryParam("type") final String type,
-            @QueryParam("projects") final List<String> projects,
+            @QueryParam(AuthorizationFilter.PROJECTS_PARAM) final List<String> projects,
             @QueryParam("maxresults") @DefaultValue(MAX_RESULTS + "") final int maxResults,
             @QueryParam("start") @DefaultValue(0 + "") final int startDocIndex
```

Comment:
now this is not behind the localhost filter and is just covered with the authorization filter?

Reply:
Yep, `LocalhostFilter` has an exemption for the `search` path.